README.md

```diff
@@ -216,6 +216,35 @@ The open and close callbacks come before and after the entire retry in the simpl
 
 Note that when there is more than one listener, they are in a list, so there is an order. In this case open will be called in the same order while onError and close will be called in reverse order.
 
+### Listeners for reflective method invocations
+
+When dealing with methods annotated with `@Retryable` or with Spring AOP intercepted methods,
+spring-retry provides the possibility to inspect in detail the method invocation within the
+`RetryListener` implementation. Such a scenario could be particularly useful when there is a need
```

Comment:
Would be great to follow *One sentence per line rule*: https://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line

Reply:
thanks for the heads up.

i've modified the documentation acordingly to your suggestion.